src/components/network/network-table.js

```diff
@@ -144,31 +358,385 @@ const NetworkTable = (props) => {
         manualTabSwitch,
     ]);
 
+    const renderTableLockIcon = useCallback(() => {
+        return <LockIcon className={classes.tableLock} />;
+    }, [classes.tableLock]);
+
+    const sortIconClassStyle = useCallback(
+        (columnSort, dataKey) => {
+            if (columnSort && columnSort.key === dataKey) {
+                return classes.activeSortArrow;
+            }
+            return classes.inactiveSortArrow;
+        },
+        [classes.activeSortArrow, classes.inactiveSortArrow]
+    );
+
+    const renderSortArrowIcon = (columnSort, dataKey) => {
+        if (columnSort && columnSort.key === dataKey && columnSort.reverse) {
+            return <ArrowUpwardIcon className={'arrow'} />;
+        }
+        return <ArrowDownwardIcon className={'arrow'} />;
+    };
+
+    const renderColumnConfigLockIcon = (value) => {
+        if (selectedColumnsNames.has(value)) {
+            if (lockedColumnsNames.has(value)) {
+                return <LockIcon className={classes.columnConfigClosedLock} />;
+            } else {
+                if (lockedColumnsNames.size < MAX_LOCKS_PER_TAB) {
+                    return (
+                        <LockOpenIcon
+                            className={classes.columnConfigOpenLock}
+                        />
+                    );
+                }
+            }
+        }
+    };
+
+    const setSort = useCallback(
```

Comment:
May be we could ask the PO whether we want to be able to return to 'not sorted' and a have a three state there (unsorted -> sorted -> sorted reverse -> not sorted) ?

Reply:
Changed it to a three state sort. It is better in the case where you start to edit something, click on the table name and "lose" your line in the 2000 rows of the table. If you can undo the sort, then your line comes back where it was and you can resume your edit.